src/components/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const one = new Big(numberOne);
+  const two = new Big(numberTwo);
+  let result;
+  switch (operation) {
+    case '+':
+      result = one.add(two);
+      break;
+    case '-':
+      result = one.sub(two);
+      break;
+    case 'X':
+      result = one.mul(two);
+      break;
+    case '÷':
+      result = one.div(two);
+      break;
+    case '%':
+      result = one.mod(two);
+      break;
```

Comment:
Please note that `%` should turn the number into a percentage based number, `e.g. 95 % == .95`